x-pack/plugins/spaces/server/routes/api/__fixtures__/create_mock_so_repository.ts

```diff
@@ -27,13 +27,14 @@ export const createMockSavedObjectsRepository = (spaces: any[] = []) => {
       if (spaces.find((s) => s.id === id)) {
         throw SavedObjectsErrorHelpers.decorateConflictError(new Error(), 'space conflict');
       }
-      return {};
+      return { id, attributes };
```

Comment:
Returning a non-empty object now to satisfy the explicit conversion in the client.